src/secureCreatives.js

```diff
@@ -74,12 +74,39 @@ function sendAdToCreative(adObject, remoteDomain, source) {
 function resizeRemoteCreative({ adUnitCode, width, height }) {
   // resize both container div + iframe
   ['div', 'iframe'].forEach(elmType => {
-    let elementStyle = getElementByAdUnit(elmType).style;
-    elementStyle.width = width + 'px';
-    elementStyle.height = height + 'px';
+    let element = getElementByAdUnit(elmType);
+    if (element) {
+      let elementStyle = element.style;
+      elementStyle.width = width + 'px';
+      elementStyle.height = height + 'px';
+    } else {
+      logWarn(`Unable to locate matching page element for adUnitCode ${adUnitCode}.  Can't resize it to ad's dimensions.  Please review setup.`);
+    }
   });
+
   function getElementByAdUnit(elmType) {
-    return document.getElementById(find(window.googletag.pubads().getSlots().filter(isSlotMatchingAdUnitCode(adUnitCode)), slot => slot)
-      .getSlotElementId()).querySelector(elmType);
+    let id = getElementIdBasedOnAdServer(adUnitCode);
+    let parentDivEle = document.getElementById(id);
+    return parentDivEle && parentDivEle.querySelector(elmType);
+  }
+
+  function getElementIdBasedOnAdServer(adUnitCode) {
+    if (window.googletag) {
+      return getDfpElementId(adUnitCode)
+    } else if (window.apntag) {
+      return getAstElementId(adUnitCode)
+    } else {
+      return adUnitCode;
+    }
+  }
+
+  function getDfpElementId(adUnitCode) {
+    return find(window.googletag.pubads().getSlots().filter(isSlotMatchingAdUnitCode(adUnitCode)), slot => slot).getSlotElementId()
+  }
+
+  function getAstElementId(adUnitCode) {
+    let apn = window.apntag;
+    let astTagIds = apn.requests && apn.requests.tags && Object.keys(apn.requests.tags);
```

Comment:
let's see if there is a way to do this without accessing `apn.request` object.

Reply:
I replaced the `apntag.requests` reference with the new API function.